app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java

```diff
@@ -138,7 +138,10 @@ public void createEvent() {
             .subscribe(createdEvent -> {
                 onSuccess.setValue("Event Created Successfully");
                 close.setValue(null);
-            }, Logger::logError));
+            }, throwable -> {
+                Logger.logError(throwable);
+                onError.setValue(throwable.getMessage());
+            }));
```

Comment:
No, handle the error and display proper message

Reply:
Done

Reply:
What about unhandled error? It should not print exceptions for unhandled errors

Reply:
Updated

Reply:
It should not show toasts for unhandled errors